src/faebryk/libs/sexp/dataclass_sexp.py

```diff
@@ -99,24 +100,36 @@ def _convert(
         if (origin := get_origin(t)) is not None:
             args = get_args(t)
             if origin is list:
-                return [_convert(_val, args[0], substack) for _val in val]
+                return [
```

Comment:
This is as a backstop to TRY load stuff. I think we could take a more delicate approach in the future, but works well for now. See https://github.com/atopile/atopile/pull/711/files#diff-02f2b52a27305bdc76f47e7e4e5e3040a17a3b9b12bcb4cb3593b245e028e6a9